src/components/views/Profile/Profile.tsx

```diff
@@ -0,0 +1,47 @@
+import auth from '@react-native-firebase/auth';
+import firestore from '@react-native-firebase/firestore';
+import {useActor} from '@xstate/react';
+import {View, Text, Button} from 'react-native';
+import {useContext} from 'react';
+import {GlobalStateContext} from '../../../components/molecules';
+
+export const Profile = () => {
+  const globalServices = useContext(GlobalStateContext);
+  //@ts-ignore
+  const [state, send] = useActor(globalServices.userService);
+  const currentUser = auth().currentUser;
+  const deleteUser = () => {
+    firestore()
+      .collection('Users')
+      .doc(currentUser?.uid)
+      .delete()
+      .then(() => {
+        currentUser?.delete();
+      })
+      .catch(err => alert(`coś się wyjebało: ${err.message}`));
+  };
+  return (
+    <View>
+      <Text>Profile</Text>
+      <Text>id:{state.context.id}</Text>
```

Comment:
masz 100% pewności, że "state.context.id" itp. nigdy nie będzie undefined czy coś? Bo jak wsadzisz tu jakiegoś undefined czy null'a to ci apke bedzie crashowac

Reply:
no teoretycznie tak, bo przy renderowaniu się tego pierwszego taba jak się zalogujesz to pobiera dane użytkownika i wrzuca je do state, a te dane powinny być bo jak zakłada se ktoś konto to tworzy od razu w bazie użytkownika z id, login i mail i to powinna być tam będą jeszcze inne dane które już mogą być i mogą nie być xD